client-ts/Microsoft.AspNetCore.SignalR.Client.TS/Transports.ts

```diff
@@ -168,6 +168,12 @@ export class LongPollingTransport implements ITransport {
     connect(url: string, requestedTransferMode: TransferMode): Promise<TransferMode> {
         this.url = url;
         this.shouldPoll = true;
+
+        let actualTransferMode =
```

Comment:
I will remove this code.

This was my failed attempt to send MessagePack over non-level-2 xhr. This won't work because the server assumes it knows transfer modes a transport supports. I opened https://github.com/aspnet/SignalR/issues/742 for this.